Merchant/Processor.ts

```diff
@@ -9,6 +9,7 @@ export interface Processor {
 	}
 	ipg?: { reference: string }
 	baboba?: { reference: string }
+	clearhaus?: { reference: string }
```

Comment:
We have to have a merchant specific key, that won't be specified in the key for our acquirer:
```suggestion
	clearhaus?: { 
	reference: string
	key: string 
	}
```